lib/rspec/core/output_wrapper.rb

```diff
@@ -3,13 +3,17 @@ module Core
     # @private
     class OutputWrapper
       # @private
-      attr_writer :output
+      attr_accessor :output
 
       # @private
       def initialize(output)
         @output = output
       end
 
+      def respond_to?(name, priv=false)
+        @output.respond_to?(name, priv)
+      end
```

Comment:
I moved the strings back to the formatter loader because it's simpler and handles all the existing behaviour, this object is just meant to be a persistent handler to the configured output, I'm happy to drop the IO methods and just use plain old method_missing, it's just that defined methods are quicker